pillar/heroku/discussions.sls

```diff
@@ -168,6 +171,7 @@ heroku:
     FEATURE_SEARCH_UI: True
     FEATURE_USE_NEW_BRANDING: True
     FEATURE_WIDGETS_UI: True
+    GA_G_TRACKING_ID: {{ env_data.GA_G_TRACKING_ID }}
```

Comment:
Kinda wish the key had a more descriptive name since we already have a `GA_TRACKING_ID`. Maybe `GTM_TRACKING_ID` which is what we have in [bootcamps](https://github.com/mitodl/salt-ops/blob/main/pillar/heroku/bootcamps.sls#L19)

Reply:
The key is GA_G_TRACKING_ID in the PR that has been merged in open. The plan is to remove GA_TRACKING_ID once we verify that we are getting the same metrics, so we won't have both keys long term